src/doc/reference.md

```diff
@@ -3064,6 +3064,17 @@ of a condition expression it expects a refutable let statement. If the value of
 expression on the right hand side of the let statement matches the pattern, the corresponding
 block will execute, otherwise flow proceeds to the first `else` block that follows.
 
+```
```

Comment:
if this example does not execute, it needs a `rust,ignore` after the triple ticks